README.md

```diff
@@ -363,6 +363,48 @@ var_dump($flat);
 // {0: 'one', 1: 'two', 0: 'three'}
 ```
 
+## More examples
+
+With the example data above, this is how you could use itertools to get all unique colors of the cars in alphabetical order:
+
+```php
+use function Zicht\Itertools\iterable;
+
+$vehicles =iterable($vehicles)
+    ->filter(equals('car', 'type')) // {[vehicle...], [vehicle...]}
+    ->map('colors') // {0: ['red', 'green', 'blue'], 1: ['blue']}
+    ->collapse() // {0: 'red', 1: 'green', 2: 'blue', 3: 'blue'}
+    ->unique() // {0: 'red', 1: 'green', 2: 'blue'}
+    ->sorted(); // {2: 'blue', 1: 'green', 0: 'red'}
+```
+
+
+Assuming the variable `vehicles` is defined in the controller that calls the template, you can achieve the same in Twig.
+
+Debugging:
+```twig
+{{ dump(vehicles
+    |filter(filtering('equals', 'car', 'type'))
+    |map('colors')
+    |collapse
+    |unique
+    |sorted)
+}}
+```
+
+Usage in a for loop:
+```twig
+{% for vehicleColor in vehicles
+    |filter(filtering('equals', 'car', 'type'))
+    |map('colors')
+    |collapse
+    |unique
+    |sorted
+%}
+    {{ vehicleColor }}
+{% endfor %}
+```
+
```

Comment:
This entire block could be moved up the file, perhaps just below where the `vehicle` data is described?  And a link to this section could be placed at the top of the readme file, because most people will want to see this